awesome_cart/awc.py

```diff
@@ -1591,7 +1591,8 @@ def cart(data=None, action=None):
 			success = False
 			msg = "Please Login to Apply Coupon"
 			awc["discounts"] = None
-			del awc["totals"]["coupon"]
+			if "coupon" in awc["totals"]:
```

Comment:
```suggestion
			awc["totals"].pop("coupon", None)
```

You can safely remove keys with that

Reply:
hm... yeah I've see those. But its way more apparent what it is that you are doing if you see the if statement checking for the key

Reply:
Amazingly I like readability over the one liner on this one...

I know shocking 😅